### PR TITLE
add a new module to configure isis on isis-view.

### PR DESCRIPTION
**Pull request by yuandongx**
##### ISSUE TYPE

- New Module Pull Request

##### ADDITIONAL INFORMATION

----test case ---
  - name: Merge isis isLevel level_1
    ce_is_is_view:
      instance_id: 3
      islevel: level_1
      state: present

```
### test log
```

changed: [10.190.121.125] => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python"
    },
    "changed": true,
    "end_state": {
        "instance": {
            "costStyle": "narrow",
            "description": null,
            "instanceId": "3",
            "isLevel": "level_1",
            "relaxSpfLimit": "false",
            "stdAutoCostEnable": "false",
            "stdAutoCostEnableCompatible": "false",
            "stdLevel1Cost": null,
            "stdLevel2Cost": null,
            "stdbandwidth": "100"
        }
    },
    "existing": {
        "instance": {
            "costStyle": "narrow",
            "description": null,
            "instanceId": "3",
            "isLevel": "level_1_2",
            "relaxSpfLimit": "false",
            "stdAutoCostEnable": "false",
            "stdAutoCostEnableCompatible": "false",
            "stdLevel1Cost": null,
            "stdLevel2Cost": null,
            "stdbandwidth": "100"
        }
    },
    "invocation": {
        "module_args": {
            "aclnum_or_name": null,
            "allow_filter": null,
            "allow_up_down": null,
            "autocostenable": null,
            "autocostenablecompatible": null,
            "avoid_learning": null,
            "bfd_min_rx": null,
            "bfd_min_tx": null,
            "bfd_multiplier_num": null,
            "cost": null,
            "cost_type": null,
            "coststyle": null,
            "defaultmode": null,
            "description": null,
            "enablelevel1tolevel2": null,
            "export_aclnumorname": null,
            "export_ipprefix": null,
            "export_policytype": null,
            "export_processid": null,
            "export_protocol": null,
            "export_routepolicyname": null,
            "host": "10.190.121.125",
            "import_aclnumorname": null,
            "import_cost": null,
            "import_ipprefix": null,
            "import_route_policy": null,
            "import_routepolicy_name": null,
            "import_routepolicyname": null,
            "import_tag": null,
            "impotr_leveltype": null,
            "inheritcost": null,
            "instance_id": 3,
            "ip_address": null,
            "ip_prefix_name": null,
            "islevel": "level_1",
            "level_type": null,
            "max_load": null,
            "mode_routepolicyname": null,
            "mode_tag": null,
            "netentity": null,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "penetration_direct": null,
            "permitibgp": null,
            "port": 10111,
            "preference_value": null,
            "processid": null,
            "protocol": null,
            "relaxspfLimit": null,
            "route_policy_name": null,
            "ssh_keyfile": null,
            "state": "present",
            "stdbandwidth": null,
            "stdlevel1cost": null,
            "stdlevel2cost": null,
            "tag": null,
            "timeout": null,
            "transport": "cli",
            "use_ssl": null,
            "username": "huawei",
            "validate_certs": null,
            "weight": null
        }
    },
    "proposed": {
        "aclnum_or_name": null,
        "allow_filter": null,
        "allow_up_down": null,
        "autocostenable": null,
        "autocostenablecompatible": null,
        "bfd_min_rx": null,
        "bfd_min_tx": null,
        "bfd_multiplier_num": null,
        "cost_type": null,
        "coststyle": null,
        "description": null,
        "enablelevel1tolevel2": null,
        "export_aclnumorname": null,
        "export_ipprefix": null,
        "export_policytype": null,
        "export_processid": null,
        "export_protocol": null,
        "export_routepolicyname": null,
        "import_aclnumorname": null,
        "import_cost": null,
        "import_ipprefix": null,
        "import_route_policy": null,
        "import_routepolicy_name": null,
        "import_routepolicyname": null,
        "import_tag": null,
        "impotr_leveltype": null,
        "inheritcost": null,
        "instance_id": 3,
        "ip_address": null,
        "ip_prefix_name": null,
        "islevel": "level_1",
        "max_load": null,
        "netentity": null,
        "penetration_direct": null,
        "permitibgp": null,
        "preference_value": null,
        "processid": null,
        "protocol": null,
        "relaxspfLimit": null,
        "route_policy_name": null,
        "state": "present",
        "stdbandwidth": null,
        "stdlevel1cost": null,
        "stdlevel2cost": null,
        "tag": null,
        "weight": null
    },
    "updates": [
        "isis 3",
        "is-level level_1"
    ]
}
META: ran handlers
META: ran handlers

PLAY RECAP *************************************************************************************
10.190.121.125             : ok=1    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0

```
### show on real device
```
[~8850_130.12-isis-3]di th
#
isis 3
 is-level level-1
 bfd all-interfaces min-tx-interval 52 min-rx-interval 51 detect-multiplier 5
 preference 1
#
return
```
### show on real device
```
[~8850_130.12-isis-3]

```
### test case
```

  - name: Merge isis stdlevel1cost 1  coststyle narrow
    ce_is_is_view:
      instance_id: 3
      coststyle: narrow
      stdlevel1cost: 1
      state: present

```
### test log
```

changed: [10.190.121.125] => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python"
    },
    "changed": true,
    "end_state": {
        "instance": {
            "costStyle": "narrow",
            "description": null,
            "instanceId": "3",
            "isLevel": "level_1",
            "relaxSpfLimit": "false",
            "stdAutoCostEnable": "false",
            "stdAutoCostEnableCompatible": "false",
            "stdLevel1Cost": "1",
            "stdLevel2Cost": null,
            "stdbandwidth": "100"
        }
    },
    "existing": {
        "instance": {
            "costStyle": "narrow",
            "description": null,
            "instanceId": "3",
            "isLevel": "level_1",
            "relaxSpfLimit": "false",
            "stdAutoCostEnable": "false",
            "stdAutoCostEnableCompatible": "false",
            "stdLevel1Cost": null,
            "stdLevel2Cost": null,
            "stdbandwidth": "100"
        }
    },
    "invocation": {
        "module_args": {
            "aclnum_or_name": null,
            "allow_filter": null,
            "allow_up_down": null,
            "autocostenable": null,
            "autocostenablecompatible": null,
            "avoid_learning": null,
            "bfd_min_rx": null,
            "bfd_min_tx": null,
            "bfd_multiplier_num": null,
            "cost": null,
            "cost_type": null,
            "coststyle": "narrow",
            "defaultmode": null,
            "description": null,
            "enablelevel1tolevel2": null,
            "export_aclnumorname": null,
            "export_ipprefix": null,
            "export_policytype": null,
            "export_processid": null,
            "export_protocol": null,
            "export_routepolicyname": null,
            "host": "10.190.121.125",
            "import_aclnumorname": null,
            "import_cost": null,
            "import_ipprefix": null,
            "import_route_policy": null,
            "import_routepolicy_name": null,
            "import_routepolicyname": null,
            "import_tag": null,
            "impotr_leveltype": null,
            "inheritcost": null,
            "instance_id": 3,
            "ip_address": null,
            "ip_prefix_name": null,
            "islevel": null,
            "level_type": null,
            "max_load": null,
            "mode_routepolicyname": null,
            "mode_tag": null,
            "netentity": null,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "penetration_direct": null,
            "permitibgp": null,
            "port": 10111,
            "preference_value": null,
            "processid": null,
            "protocol": null,
            "relaxspfLimit": null,
            "route_policy_name": null,
            "ssh_keyfile": null,
            "state": "present",
            "stdbandwidth": null,
            "stdlevel1cost": 1,
            "stdlevel2cost": null,
            "tag": null,
            "timeout": null,
            "transport": "cli",
            "use_ssl": null,
            "username": "huawei",
            "validate_certs": null,
            "weight": null
        }
    },
    "proposed": {
        "aclnum_or_name": null,
        "allow_filter": null,
        "allow_up_down": null,
        "autocostenable": null,
        "autocostenablecompatible": null,
        "bfd_min_rx": null,
        "bfd_min_tx": null,
        "bfd_multiplier_num": null,
        "cost_type": null,
        "coststyle": "narrow",
        "description": null,
        "enablelevel1tolevel2": null,
        "export_aclnumorname": null,
        "export_ipprefix": null,
        "export_policytype": null,
        "export_processid": null,
        "export_protocol": null,
        "export_routepolicyname": null,
        "import_aclnumorname": null,
        "import_cost": null,
        "import_ipprefix": null,
        "import_route_policy": null,
        "import_routepolicy_name": null,
        "import_routepolicyname": null,
        "import_tag": null,
        "impotr_leveltype": null,
        "inheritcost": null,
        "instance_id": 3,
        "ip_address": null,
        "ip_prefix_name": null,
        "islevel": null,
        "max_load": null,
        "netentity": null,
        "penetration_direct": null,
        "permitibgp": null,
        "preference_value": null,
        "processid": null,
        "protocol": null,
        "relaxspfLimit": null,
        "route_policy_name": null,
        "state": "present",
        "stdbandwidth": null,
        "stdlevel1cost": 1,
        "stdlevel2cost": null,
        "tag": null,
        "weight": null
    },
    "updates": [
        "isis 3",
        "circuit-cost 1 level-1"
    ]
}
META: ran handlers
META: ran handlers

PLAY RECAP *************************************************************************************
10.190.121.125             : ok=1    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0

```
### show on real device
```
[~8850_130.12-isis-3]di th
#
isis 3
 is-level level-1
 bfd all-interfaces min-tx-interval 52 min-rx-interval 51 detect-multiplier 5
 circuit-cost 1 level-1
 preference 1
#
return
```
### show on real device
```
[~8850_130.12-isis-3]

```
### test case
```

  - name: Merge isis nexthop weight 1
    ce_is_is_view:
      instance_id: 3
      ip_address: 1.2.3.5
      weight: 1
      state: present

```
### test log
```

changed: [10.190.121.125] => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python"
    },
    "changed": true,
    "end_state": {
        "instance": {
            "ipAddress": "1.2.3.5",
            "weight": "1"
        }
    },
    "existing": {
        "instance": {}
    },
    "invocation": {
        "module_args": {
            "aclnum_or_name": null,
            "allow_filter": null,
            "allow_up_down": null,
            "autocostenable": null,
            "autocostenablecompatible": null,
            "avoid_learning": null,
            "bfd_min_rx": null,
            "bfd_min_tx": null,
            "bfd_multiplier_num": null,
            "cost": null,
            "cost_type": null,
            "coststyle": null,
            "defaultmode": null,
            "description": null,
            "enablelevel1tolevel2": null,
            "export_aclnumorname": null,
            "export_ipprefix": null,
            "export_policytype": null,
            "export_processid": null,
            "export_protocol": null,
            "export_routepolicyname": null,
            "host": "10.190.121.125",
            "import_aclnumorname": null,
            "import_cost": null,
            "import_ipprefix": null,
            "import_route_policy": null,
            "import_routepolicy_name": null,
            "import_routepolicyname": null,
            "import_tag": null,
            "impotr_leveltype": null,
            "inheritcost": null,
            "instance_id": 3,
            "ip_address": "1.2.3.5",
            "ip_prefix_name": null,
            "islevel": null,
            "level_type": null,
            "max_load": null,
            "mode_routepolicyname": null,
            "mode_tag": null,
            "netentity": null,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "penetration_direct": null,
            "permitibgp": null,
            "port": 10111,
            "preference_value": null,
            "processid": null,
            "protocol": null,
            "relaxspfLimit": null,
            "route_policy_name": null,
            "ssh_keyfile": null,
            "state": "present",
            "stdbandwidth": null,
            "stdlevel1cost": null,
            "stdlevel2cost": null,
            "tag": null,
            "timeout": null,
            "transport": "cli",
            "use_ssl": null,
            "username": "huawei",
            "validate_certs": null,
            "weight": 1
        }
    },
    "proposed": {
        "aclnum_or_name": null,
        "allow_filter": null,
        "allow_up_down": null,
        "autocostenable": null,
        "autocostenablecompatible": null,
        "bfd_min_rx": null,
        "bfd_min_tx": null,
        "bfd_multiplier_num": null,
        "cost_type": null,
        "coststyle": null,
        "description": null,
        "enablelevel1tolevel2": null,
        "export_aclnumorname": null,
        "export_ipprefix": null,
        "export_policytype": null,
        "export_processid": null,
        "export_protocol": null,
        "export_routepolicyname": null,
        "import_aclnumorname": null,
        "import_cost": null,
        "import_ipprefix": null,
        "import_route_policy": null,
        "import_routepolicy_name": null,
        "import_routepolicyname": null,
        "import_tag": null,
        "impotr_leveltype": null,
        "inheritcost": null,
        "instance_id": 3,
        "ip_address": "1.2.3.5",
        "ip_prefix_name": null,
        "islevel": null,
        "max_load": null,
        "netentity": null,
        "penetration_direct": null,
        "permitibgp": null,
        "preference_value": null,
        "processid": null,
        "protocol": null,
        "relaxspfLimit": null,
        "route_policy_name": null,
        "state": "present",
        "stdbandwidth": null,
        "stdlevel1cost": null,
        "stdlevel2cost": null,
        "tag": null,
        "weight": 1
    },
    "updates": [
        "isis 3",
        "nexthop 1.2.3.5 weight 1"
    ]
}
META: ran handlers
META: ran handlers

PLAY RECAP *************************************************************************************
10.190.121.125             : ok=1    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0

```
### show on real device
```
[~8850_130.12-isis-3]di th
#
isis 3
 is-level level-1
 bfd all-interfaces min-tx-interval 52 min-rx-interval 51 detect-multiplier 5
 circuit-cost 1 level-1
 nexthop 1.2.3.5 weight 1
 preference 1
#
return
```
### show on real device
```
[~8850_130.12-isis-3]

```
### test case
```

  - name: set isis route penetration level2-level1  aclnum 2001
    ce_is_is_view:
      instance_id: 3
      penetration_direct: level2-level1
      aclnum_or_name: 2001
      tag: 3
      allow_filter: true
      state: present

```
### test log
```

changed: [10.190.121.125] => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python"
    },
    "changed": true,
    "end_state": {
        "instance": {
            "aclNumOrName": "2001",
            "allowFilter": "true",
            "allowUpdown": "false",
            "ipPrefix": null,
            "routePolicyName": null,
            "tag": "3"
        }
    },
    "existing": {
        "instance": {}
    },
    "invocation": {
        "module_args": {
            "aclnum_or_name": "2001",
            "allow_filter": true,
            "allow_up_down": null,
            "autocostenable": null,
            "autocostenablecompatible": null,
            "avoid_learning": null,
            "bfd_min_rx": null,
            "bfd_min_tx": null,
            "bfd_multiplier_num": null,
            "cost": null,
            "cost_type": null,
            "coststyle": null,
            "defaultmode": null,
            "description": null,
            "enablelevel1tolevel2": null,
            "export_aclnumorname": null,
            "export_ipprefix": null,
            "export_policytype": null,
            "export_processid": null,
            "export_protocol": null,
            "export_routepolicyname": null,
            "host": "10.190.121.125",
            "import_aclnumorname": null,
            "import_cost": null,
            "import_ipprefix": null,
            "import_route_policy": null,
            "import_routepolicy_name": null,
            "import_routepolicyname": null,
            "import_tag": null,
            "impotr_leveltype": null,
            "inheritcost": null,
            "instance_id": 3,
            "ip_address": null,
            "ip_prefix_name": null,
            "islevel": null,
            "level_type": null,
            "max_load": null,
            "mode_routepolicyname": null,
            "mode_tag": null,
            "netentity": null,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "penetration_direct": "level2-level1",
            "permitibgp": null,
            "port": 10111,
            "preference_value": null,
            "processid": null,
            "protocol": null,
            "relaxspfLimit": null,
            "route_policy_name": null,
            "ssh_keyfile": null,
            "state": "present",
            "stdbandwidth": null,
            "stdlevel1cost": null,
            "stdlevel2cost": null,
            "tag": 3,
            "timeout": null,
            "transport": "cli",
            "use_ssl": null,
            "username": "huawei",
            "validate_certs": null,
            "weight": null
        }
    },
    "proposed": {
        "aclnum_or_name": "2001",
        "allow_filter": true,
        "allow_up_down": null,
        "autocostenable": null,
        "autocostenablecompatible": null,
        "bfd_min_rx": null,
        "bfd_min_tx": null,
        "bfd_multiplier_num": null,
        "cost_type": null,
        "coststyle": null,
        "description": null,
        "enablelevel1tolevel2": null,
        "export_aclnumorname": null,
        "export_ipprefix": null,
        "export_policytype": null,
        "export_processid": null,
        "export_protocol": null,
        "export_routepolicyname": null,
        "import_aclnumorname": null,
        "import_cost": null,
        "import_ipprefix": null,
        "import_route_policy": null,
        "import_routepolicy_name": null,
        "import_routepolicyname": null,
        "import_tag": null,
        "impotr_leveltype": null,
        "inheritcost": null,
        "instance_id": 3,
        "ip_address": null,
        "ip_prefix_name": null,
        "islevel": null,
        "max_load": null,
        "netentity": null,
        "penetration_direct": "level2-level1",
        "permitibgp": null,
        "preference_value": null,
        "processid": null,
        "protocol": null,
        "relaxspfLimit": null,
        "route_policy_name": null,
        "state": "present",
        "stdbandwidth": null,
        "stdlevel1cost": null,
        "stdlevel2cost": null,
        "tag": 3,
        "weight": null
    },
    "updates": [
        "isis 3",
        "import-route isis level-2 into level-1 filter-policy acl-name 2001 tag 3 direct allow-filter-policy"
    ]
}
META: ran handlers
META: ran handlers

PLAY RECAP *************************************************************************************
10.190.121.125             : ok=1    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0

```
### show on real device
```
[~8850_130.12-isis-3]di th
#
isis 3
 is-level level-1
 bfd all-interfaces min-tx-interval 52 min-rx-interval 51 detect-multiplier 5
 circuit-cost 1 level-1
 import-route isis level-2 into level-1 tag 3 filter-policy 2001 direct allow-filter-policy
 nexthop 1.2.3.5 weight 1
 preference 1
#
return
```
### show on real device
```
[~8850_130.12-isis-3]

```
### test case
```

  - name: set isis import route
    ce_is_is_view:
      instance_id: 3
      protocol: rip
      cost_type: external
      processid: 1
      import_cost: 2
      import_tag: 1
      impotr_leveltype: level_1
      state: present

```
### test log
```

changed: [10.190.121.125] => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python"
    },
    "changed": true,
    "end_state": {
        "instance": {
            "cost": "2",
            "costType": "external",
            "inheritCost": "false",
            "levelType": "level_2",
            "permitIbgp": "false",
            "policyType": "noType",
            "processId": "1",
            "protocol": "rip",
            "routePolicyName": null,
            "tag": "1"
        }
    },
    "existing": {
        "instance": {}
    },
    "invocation": {
        "module_args": {
            "aclnum_or_name": null,
            "allow_filter": null,
            "allow_up_down": null,
            "autocostenable": null,
            "autocostenablecompatible": null,
            "avoid_learning": null,
            "bfd_min_rx": null,
            "bfd_min_tx": null,
            "bfd_multiplier_num": null,
            "cost": null,
            "cost_type": "external",
            "coststyle": null,
            "defaultmode": null,
            "description": null,
            "enablelevel1tolevel2": null,
            "export_aclnumorname": null,
            "export_ipprefix": null,
            "export_policytype": null,
            "export_processid": null,
            "export_protocol": null,
            "export_routepolicyname": null,
            "host": "10.190.121.125",
            "import_aclnumorname": null,
            "import_cost": 2,
            "import_ipprefix": null,
            "import_route_policy": null,
            "import_routepolicy_name": null,
            "import_routepolicyname": null,
            "import_tag": 1,
            "impotr_leveltype": "level_1",
            "inheritcost": null,
            "instance_id": 3,
            "ip_address": null,
            "ip_prefix_name": null,
            "islevel": null,
            "level_type": null,
            "max_load": null,
            "mode_routepolicyname": null,
            "mode_tag": null,
            "netentity": null,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "penetration_direct": null,
            "permitibgp": null,
            "port": 10111,
            "preference_value": null,
            "processid": 1,
            "protocol": "rip",
            "relaxspfLimit": null,
            "route_policy_name": null,
            "ssh_keyfile": null,
            "state": "present",
            "stdbandwidth": null,
            "stdlevel1cost": null,
            "stdlevel2cost": null,
            "tag": null,
            "timeout": null,
            "transport": "cli",
            "use_ssl": null,
            "username": "huawei",
            "validate_certs": null,
            "weight": null
        }
    },
    "proposed": {
        "aclnum_or_name": null,
        "allow_filter": null,
        "allow_up_down": null,
        "autocostenable": null,
        "autocostenablecompatible": null,
        "bfd_min_rx": null,
        "bfd_min_tx": null,
        "bfd_multiplier_num": null,
        "cost_type": "external",
        "coststyle": null,
        "description": null,
        "enablelevel1tolevel2": null,
        "export_aclnumorname": null,
        "export_ipprefix": null,
        "export_policytype": null,
        "export_processid": null,
        "export_protocol": null,
        "export_routepolicyname": null,
        "import_aclnumorname": null,
        "import_cost": 2,
        "import_ipprefix": null,
        "import_route_policy": null,
        "import_routepolicy_name": null,
        "import_routepolicyname": null,
        "import_tag": 1,
        "impotr_leveltype": "level_1",
        "inheritcost": null,
        "instance_id": 3,
        "ip_address": null,
        "ip_prefix_name": null,
        "islevel": null,
        "max_load": null,
        "netentity": null,
        "penetration_direct": null,
        "permitibgp": null,
        "preference_value": null,
        "processid": 1,
        "protocol": "rip",
        "relaxspfLimit": null,
        "route_policy_name": null,
        "state": "present",
        "stdbandwidth": null,
        "stdlevel1cost": null,
        "stdlevel2cost": null,
        "tag": null,
        "weight": null
    },
    "updates": [
        "isis 3",
        "import-route rip 1 cost-type external cost 2 tag 1 level-1"
    ]
}
META: ran handlers
META: ran handlers

PLAY RECAP *************************************************************************************
10.190.121.125             : ok=1    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0

```
### show on real device
```
[~8850_130.12-isis-3]di th
#
isis 3
 is-level level-1
 bfd all-interfaces min-tx-interval 52 min-rx-interval 51 detect-multiplier 5
 circuit-cost 1 level-1
 import-route isis level-2 into level-1 tag 3 filter-policy 2001 direct allow-filter-policy
 import-route rip 1 cost 2 tag 1
 nexthop 1.2.3.5 weight 1
 preference 1
#
return
```
### show on real device
```
[~8850_130.12-isis-3]

```
### test case
```

  - name: set isis bfd Conversation rx 100 tx 100 num 10
    ce_is_is_view:
      instance_id: 3
      bfd_min_rx: 100
      bfd_min_tx: 100
      bfd_multiplier_num: 10
      state: present

```
### test log
```

changed: [10.190.121.125] => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python"
    },
    "changed": true,
    "end_state": {
        "instance": {
            "addressFamily": "afIpv4",
            "bfdMinRx": "100",
            "bfdMinTx": "100",
            "bfdMultNum": "10",
            "mtId": "0"
        }
    },
    "existing": {
        "instance": {
            "addressFamily": "afIpv4",
            "bfdMinRx": "51",
            "bfdMinTx": "52",
            "bfdMultNum": "5",
            "mtId": "0"
        }
    },
    "invocation": {
        "module_args": {
            "aclnum_or_name": null,
            "allow_filter": null,
            "allow_up_down": null,
            "autocostenable": null,
            "autocostenablecompatible": null,
            "avoid_learning": null,
            "bfd_min_rx": 100,
            "bfd_min_tx": 100,
            "bfd_multiplier_num": 10,
            "cost": null,
            "cost_type": null,
            "coststyle": null,
            "defaultmode": null,
            "description": null,
            "enablelevel1tolevel2": null,
            "export_aclnumorname": null,
            "export_ipprefix": null,
            "export_policytype": null,
            "export_processid": null,
            "export_protocol": null,
            "export_routepolicyname": null,
            "host": "10.190.121.125",
            "import_aclnumorname": null,
            "import_cost": null,
            "import_ipprefix": null,
            "import_route_policy": null,
            "import_routepolicy_name": null,
            "import_routepolicyname": null,
            "import_tag": null,
            "impotr_leveltype": null,
            "inheritcost": null,
            "instance_id": 3,
            "ip_address": null,
            "ip_prefix_name": null,
            "islevel": null,
            "level_type": null,
            "max_load": null,
            "mode_routepolicyname": null,
            "mode_tag": null,
            "netentity": null,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "penetration_direct": null,
            "permitibgp": null,
            "port": 10111,
            "preference_value": null,
            "processid": null,
            "protocol": null,
            "relaxspfLimit": null,
            "route_policy_name": null,
            "ssh_keyfile": null,
            "state": "present",
            "stdbandwidth": null,
            "stdlevel1cost": null,
            "stdlevel2cost": null,
            "tag": null,
            "timeout": null,
            "transport": "cli",
            "use_ssl": null,
            "username": "huawei",
            "validate_certs": null,
            "weight": null
        }
    },
    "proposed": {
        "aclnum_or_name": null,
        "allow_filter": null,
        "allow_up_down": null,
        "autocostenable": null,
        "autocostenablecompatible": null,
        "bfd_min_rx": 100,
        "bfd_min_tx": 100,
        "bfd_multiplier_num": 10,
        "cost_type": null,
        "coststyle": null,
        "description": null,
        "enablelevel1tolevel2": null,
        "export_aclnumorname": null,
        "export_ipprefix": null,
        "export_policytype": null,
        "export_processid": null,
        "export_protocol": null,
        "export_routepolicyname": null,
        "import_aclnumorname": null,
        "import_cost": null,
        "import_ipprefix": null,
        "import_route_policy": null,
        "import_routepolicy_name": null,
        "import_routepolicyname": null,
        "import_tag": null,
        "impotr_leveltype": null,
        "inheritcost": null,
        "instance_id": 3,
        "ip_address": null,
        "ip_prefix_name": null,
        "islevel": null,
        "max_load": null,
        "netentity": null,
        "penetration_direct": null,
        "permitibgp": null,
        "preference_value": null,
        "processid": null,
        "protocol": null,
        "relaxspfLimit": null,
        "route_policy_name": null,
        "state": "present",
        "stdbandwidth": null,
        "stdlevel1cost": null,
        "stdlevel2cost": null,
        "tag": null,
        "weight": null
    },
    "updates": [
        "isis 3",
        "bfd all-interfaces enable",
        "bfd all-interfaces min-rx-interval 100 min-tx-interval 100 detect-multiplier 10"
    ]
}
META: ran handlers
META: ran handlers

PLAY RECAP *************************************************************************************
10.190.121.125             : ok=1    changed=1    unreachable=0    failed=0    skipped=0    resc                                                                                                 ued=0    ignored=0

```
### show on real device
```
[~8850_130.12-isis-3]di th
#
isis 3
 is-level level-1
 bfd all-interfaces min-tx-interval 100 min-rx-interval 100 detect-multiplier 10
 circuit-cost 1 level-1
 import-route isis level-2 into level-1 tag 3 filter-policy 2001 direct allow-filter-policy
 import-route rip 1 cost 2 tag 1
 nexthop 1.2.3.5 weight 1
 preference 1
#
return
```
### show on real device
```
[~8850_130.12-isis-3]

```

```

### CODE REVIEW

**Comment by xuxiaowei0512** [comment]
-label bug

File: lib/ansible/modules/network/cloudengine/ce_is_is_view.py [hunk 1]
```diff
@@ -0,0 +1,1964 @@
...
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+---
+module: ce_is_is_view
+version_added: "2.10"
+author: xuxiaowei0512 (@CloudEngine-Ansible)
+short_description: Manages isis view configuration on HUAWEI CloudEngine devices.
+description:
+    - Manages  isis process id, creates a isis instance id or deletes a process id
```
Comment (yuandongxx) [inline lib/ansible/modules/network/cloudengine/ce_is_is_view.py]: Manages isis process id

**Review by yuandongxx — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/network/cloudengine/ce_is_is_view.py (hunk 1 above)
Comment (yuandongx) [reply lib/ansible/modules/network/cloudengine/ce_is_is_view.py]: Done

**Comment by xuxiaowei0512** [comment]
read_for_review

**Comment by xuxiaowei0512** [comment]
ready_for_review

**Comment by hwDCN** [comment]
shipit

**Comment by hwDCN** [comment]
shipit

**Comment by hwDCN** [comment]
+1

**Comment by hwDCN** [comment]
lgtm

**Comment by gundalow** [comment]
rebuild_merge

**Comment by xuxiaowei0512** [comment]
@gundalow  Thank you very much!

**Pull request merged**